tool/tsh/common/tsh.go

```diff
@@ -801,6 +846,11 @@ func Run(ctx context.Context, args []string, opts ...CliOption) error {
 	ssh.Flag("log-dir", "Directory to log separated command output, when executing on multiple nodes. If set, output from each node will also be labeled in the terminal.").StringVar(&cf.SSHLogDir)
 	ssh.Flag("no-resume", "Disable SSH connection resumption").Envar(noResumeEnvVar).BoolVar(&cf.DisableSSHResumption)
 	ssh.Flag("relogin", "Permit performing an authentication attempt on a failed command").Default("true").BoolVar(&cf.Relogin)
+	// The following flags are OpenSSH compatibility flags. They are not
+	// implemented in Kingpin but are registered here to inform developers that
+	// they are being used. Their behavior is implemented in 000000.
+	ssh.Flag(uuid.New().String(), "").Hidden().Short('T').StringVar(nil)
```

Comment:
Why use a UUID here?

Reply:
I could do a hard coded flag and try to give a strange prefix so it doesn't collide with a real flag, but figured it's easier to just use a UUID as a flag name.